README.md

```diff
@@ -0,0 +1,93 @@
+# Ionic Tabs Template - Upgraded
+
+This project provides an extension to the tabs template that is provided by Ionic. It provides the following additional functionalities.
+
+- Basic desktop compatible UI
+- Cypress testing
+- WCAG 2.2 AA compliance
+- Project formatting (using [Prettier](https://prettier.io/))
+
+Note: the template retains all provisions afforded by the original template as provided by Ionic.
+
+## How to best use this template
+
+This template provides some additional essential elements to help VueJS, Ionic developers get started on their hybrid projects easier and faster.
```

Comment:
James suggestion: "We built this because Ionic, out of the box is not desktop responsive."